test/parallel/test-tls-getcipher.js

```diff
@@ -25,6 +25,7 @@ server.listen(common.PORT, '127.0.0.1', function() {
   var client = tls.connect({
     host: '127.0.0.1',
     port: common.PORT,
+    ciphers: cipher_list.join(':'),
```

Comment:
I see now it's a bug in the test.  Can you perhaps break this out into a separate commit that precedes the main one?